frontend/src/modules/EditZing/Components/StudentCard.tsx

```diff
@@ -57,13 +101,12 @@ const StudentCard = ({
             fontFamily: 'Montserrat',
             fontWeight: '700',
             fontSize: '14',
-            boxShadow: isHovering
-              ? '4px 4px 8px rgba(0, 0, 0, 0.3)'
-              : '0px 2px 5px rgba(205, 156, 242, 0.2)',
+            boxShadow:
+              isHovering && !selected
+                ? '4px 4px 8px rgba(0, 0, 0, 0.3)'
+                : '0px 2px 5px rgba(205, 156, 242, 0.2)',
```

Comment:
Adding `transition: 'box-shadow 0.1s'` is enough to make the transition speed match that of groups